datafusion/functions-aggregate/src/min_max/min_max_bytes.rs

```diff
@@ -338,6 +338,10 @@ impl GroupsAccumulator for MinMaxBytesAccumulator {
 /// This is a heuristic to avoid allocating too many small buffers
 fn capacity_to_view_block_size(data_capacity: usize) -> u32 {
     let max_block_size = 2 * 1024 * 1024;
+    // Avoid block size equal to zero when calling `with_fixed_block_size()`.
+    if data_capacity == 0 {
+        return 1;
```

Comment:
The `data_capacity` might be zero and results in aggregation fuzz tests panicked with message "Block size must be greater than 0".

So I modify this function to ensure that the block size would not be 0 in this case. But I'm not totally sure if this is a bug...

Reply:
I bet there is something / somewhere that is passing in an empty batch -- and a small optimization might be to avoid doing so. 

Do you happen to have the stack trace still around?